aspnetcore/security/authentication/configure-jwt-bearer-authentication.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Configure JWT bearer authentication in ASP.NET Core
+author: damienbod
+description: Learn how to set up JWT bearer authentication in an ASP.NET Core app.
+monikerRange: '>= aspnetcore-8.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 12/7/2024
+uid: security/authentication/configure-jwt-bearer-authentication
+---
+# Configure JWT bearer authentication in ASP.NET Core
+
+By [Damien Bowden](https://github.com/damienbod)
+
+JWT (JSON Web Token) Bearer Authentication is commonly utilized for APIs. While it operates similarly to cookie authentication, instead of a cookie, the server issues a JWT or tokens upon a successful authentication. A JWT is a self-contained token that encapsulates information for an API resource or a client. The client which requested the JWT can request data from an API resource using the Authorization header and a bearer token.
+
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authentication/configure-jwt-bearer-authentication/sample/JwtBearer)
+
+This article covers the following areas:
+
+* Token types
+* Using JWT tokens to secure an APIs
+* How OIDC/OAuth fits into this?
+* Implementing JWT bearer token authentication
+* Recommended approaches to create a JWT
+
+## Authentication v Authorization
+
+Bearer tokens are often treated as an authentication mechanism, but in practice, the tokens are used primarily for authorization. Authentication occurs when a user first requests an access token in the UI layer. When the token is presented to an API, the process is more accurately described as *delegated authorization*. Because the token itself doesn't reveal who or what obtained it or how it was issued, the API can only confirm whether the token is valid for calling the requested resource.
+
```

Comment:
```suggestion
```

I should have asked for this earlier, but can we remove this section? I don't know who this helps. To me it only seems to muddy the waters. Bearer tokens are a critical to authentication when you're using the `JwtBearerHandler`, but in the end it's just a trusted bag of claims much like a cookie.

It sounds like you're trying to convey that, unlike with cookies, it's okay for a web application to forward JWT access tokens to other services, but that's already well covered in the "Downstream APIs" section. I don't think this should be the first section after the intro. It makes far more sense to lead with "Token Types".